components/formats-bsd/src/loci/formats/UpgradeChecker.java

```diff
@@ -85,7 +85,7 @@
    * Location of the JAR artifacts for the stable releases.
    */
   public static final String STABLE_BUILD =
-    "http://downloads.openmicroscopy.org/bio-formats/" +  STABLE_VERSION + "/artifacts/";
+    "http://downloads.openmicroscopy.org/bio-formats/";
```

Comment:
For anyone consuming this field, this would be a breaking change. Could we use the unversioned https://downloads.openmicroscopy.org/bio-formats/latest/artifacts/ URL as a replacement and deprecate the static `STABLE_VERSION` field?

If we publish using GitHub artifacts, I assume this will be replaced by the appropriate URL?

Reply:
I think that's what I had something got lost when cleaning up stuff